pygromos/files/gromos_system/ff/openforcefield2gromos.py

```diff
@@ -317,8 +250,9 @@ def convertVdW(self):
                 C6 = 2 * epsilon * (rmin**6)
                 C12 = epsilon * (rmin**12)
                 IACname = force.id
-                self.gromosTop.add_new_atom(ATNM=ATNM, MRES=MRES, PANM=PANM, IAC=IAC, MASS=MASS, CG=CG, CGC=CGC, INE=INE, INE14=INE14, C6=C6, C12=C12, IACname=IACname)
+                self.gromosTop.add_new_atom(ATNM=ATNM, MRES=MRES, PANM=PANM, IAC=IAC, MASS=MASS, CG=CG, CGC=CGC, INE=INE, INE14=INE14, C6=C6, C12=C12, CS6=0.5*C6, CS12=0.5*C12, IACname=IACname)
```

Comment:
ups where does the 0.5 come from ? documentation

Reply:
the 0.5 is the factor for the 1-4 LJ parameters. It's the same in GROMOS FF and OpenFF

Reply:
Do you think it requires documentation here in the code? @SchroederB